test-snapshots/snapshots-output/Snapshot.Constructor.ss

```diff
@@ -0,0 +1,47 @@
+#!r6rs
+#!chezscheme
+(library
+  (Snapshot.Constructor lib)
+  (export
+    Just*
+    Just-value0
+    Just?
+    Nil
+    Nil?
+    Node
+    Node*
+    Node-value0
+    Node-value1
+    Node-value2
+    Node?
+    Nothing
+    Nothing?)
+  (import
+    (prefix (chezscheme) scm:)
+    (prefix (_Chez_Runtime lib) rt:))
```

Comment:
Something to think about later, but if we aren't using anything from the runtime library, should we import it in the generated output?

Reply:
Tracked in https://github.com/purescm/purescm/issues/93